packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -67,14 +69,8 @@ export class Runner {
             return;
         }
 
-        const scanStartedTimestamp: number = Date.now();
-        const scanSubmittedTimestamp: number = this.guidGenerator.getGuidTimestamp(scanMetadata.id).getTime();
-
-        this.logger.trackEvent('ScanRequestRunning', undefined, { runningScanRequests: 1 });
-        this.logger.trackEvent('ScanTaskStarted', undefined, {
-            scanWaitTime: (scanStartedTimestamp - scanSubmittedTimestamp) / 1000,
-            startedScanTasks: 1,
-        });
+        const scanSubmittedTimestamp = this.guidGenerator.getGuidTimestamp(scanMetadata.id);
```

Comment:
Is there a reason creating the timestamp didn't get brought into the telemetry manager?